docs/pages/deploy/send-over-the-air-updates.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Send over-the-air updates
+description: Learn how to send over-the-air updates to push critical bug fixes and improvements to your users.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+You can send over-the-air updates containing critical bug fixes and improvements to your users.
+
+## Get started
+
+> If you've published [previews](/review/share-previews-with-your-team/) or created a [build](/deploy/build-project/) before, you may have already set up updates.
+
+To set up updates, run the following [EAS CLI](/develop/tools/#eas-cli) command:
+
+<Terminal cmd={['$ eas update:configure']} />
+
+Once the command completes, you'll need to make new builds.
```

Comment:
```suggestion
After the command completes, you'll need to make new builds before moving to the next section.
```

Let's add the context about creating the build before taking the next step here.